github/search.go

```diff
@@ -54,6 +54,9 @@ type SearchOptions struct {
 	// Whether to retrieve text match metadata with a query
 	TextMatch bool `url:"-"`
 
+	// Whether to enable advanced search for issues
+	AdvancedSearch bool `url:"advanced_search,omitempty"`
```

Comment:
Whups, I'm sorry, please make this change before we can merge
and then run step 4 as described in CONTRIBUTING.md and push the changes to this PR. Thank you!

```suggestion
	AdvancedSearch *bool `url:"advanced_search,omitempty"`
```